src/LiveComponent/src/LiveComponentHydrator.php

```diff
@@ -28,7 +28,7 @@
 final class LiveComponentHydrator
 {
     private const CHECKSUM_KEY = '_checksum';
-    private const EXPOSED_PROP_KEY = 'id';
+    private const EXPOSED_PROP_KEY = '_id';
```

Comment:
This doesn't mess up the frontend does it?

Reply:
I don't believe so